src/draw/opengles/lv_draw_opengles.c

```diff
@@ -570,8 +570,21 @@ static void execute_drawing(lv_draw_opengles_unit_t * u)
                 GL_CALL(glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, target_texture, 0));
             }
 
-            lv_opengles_viewport(0, 0, targ_tex_w, targ_tex_h);
-            lv_opengles_render_fill(fill_dsc->color, &fill_area, fill_dsc->opa, targ_tex_w, targ_tex_h);
+            if(fill_dsc->opa == LV_OPA_100) {
```

Comment:
```suggestion
            if(fill_dsc->opa >= LV_OPA_MAX) {
```
Opa max is 253 which is slightly lower than 255